pvlib/clearsky.py

```diff
@@ -750,20 +883,22 @@ def detect_clearsky(measured, clearsky, times, window_length,
         c2 = np.abs(meas_max - alpha*clear_max) < max_diff
         c3 = (line_diff > lower_line_length) & (line_diff < upper_line_length)
         c4 = meas_slope_nstd < var_diff
-        c5 = np.max(np.abs(meas_slope -
-                           alpha * clear_slope), axis=0) < slope_dev
+        c5 = _calc_slope_max_diff(
+            meas, alpha * clear, samples_per_window, 'center') < slope_dev
```

Comment:
nit: `alpha * clear` is computed here and on line 877, so use `scaled_clear = alpha * clear` or similar at the top of the loop